drainer/sync/mysql.go

```diff
@@ -15,6 +15,7 @@ package sync
 
 import (
 	"database/sql"
+	"github.com/pingcap/tidb-binlog/drainer/loopbacksync"
```

Comment:
can you use goimports to format the code?
It can also help you add or remove the import path automatically.